src/Xamarin.Android.Build.Tasks/Tasks/NdkUtilsOld.cs

```diff
@@ -0,0 +1,323 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using Java.Interop.Tools.Diagnostics;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+	public static class NdkUtilOld {
```

Comment:
Just a question... Should we keep `NdkUtilOld` around at all?

Reply:
Using the older NDK version in the name might be better than `Old` then, such as `NdkUtil16` or something.

Or even just a `<summary/>` command explaining it would be good 👍

Reply:
The older NDK version is anything bellow 19 - thus the suffix "Old"...